rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -246,6 +250,51 @@ def _decode_lending_pool_events(self, context: DecoderContext) -> DecodingOutput
 
         return DEFAULT_DECODING_OUTPUT
 
+    def _maybe_enrich_liquidations(
+            self,
+            context: EnricherContext,
+    ) -> TransferEnrichmentOutput:
+        """
+        Process AAVE v2 liquidations. When a liquidation happens the user returns the debt token
+        and part of the collateral deposited is lost too. Those two events happen as transfers in
+        a transaction started by the liquidator.
+        """
+        for tx_log in context.all_logs:
+            if (
+                tx_log.topics[0] != LIQUIDATION_CALL or
+                tx_log.topics[0] != LIQUIDATION_CALL and self.base.is_tracked(hex_or_bytes_to_address(context.tx_log.topics[3])) is False  # noqa: E501
+            ):
+                continue
+
+            event = context.event
+            raw_debt_to_cover = hex_or_bytes_to_int(tx_log.data[:32])
+            raw_liquidated_amount = hex_or_bytes_to_int(tx_log.data[32:64])
+            liquidator = hex_or_bytes_to_address(tx_log.data[64:96])
+            asset = event.asset.resolve_to_evm_token()
+            if event.event_type == HistoryEventType.SPEND:
```

Comment:
I would move this condition `if event.event_type == HistoryEventType.SPEND` upper and early-exit since there are no other branches anyway